### PR TITLE
refactor(@schematics/update): use npm-registry-client to get data

### PR DESCRIPTION
**Pull request by clydin**

### CODE REVIEW

File: packages/schematics/update/update/npm.ts [hunk 1]
```diff
@@ -46,23 +44,22 @@ export function getNpmPackageJson(
...
-          subject.error(err);
-        }
-      });
-      response.on('error', err => subject.error(err));
+    const client = new RegistryClient({
+
+    });
+    client.log.level = 'silent';
+    const params = {
+      timeout: 1000,
```
Comment (hansl) [inline packages/schematics/update/update/npm.ts]: Timeout needs to be larger.

File: packages/schematics/update/update/npm.ts [hunk 2]
```diff
@@ -46,23 +44,22 @@ export function getNpmPackageJson(
...
-          const json = JSON.parse(data);
-          json.requestedName = packageName;
-          subject.next(json as NpmRepositoryPackageJson);
-          subject.complete();
-        } catch (err) {
-          subject.error(err);
-        }
-      });
-      response.on('error', err => subject.error(err));
+    const client = new RegistryClient({
```
Comment (hansl) [inline packages/schematics/update/update/npm.ts]: Are there any proxy information to be passed in?

File: packages/schematics/update/update/npm.ts (hunk 1 above)
Comment (hansl) [reply packages/schematics/update/update/npm.ts]: I'd say 30 seconds is fine. All requests are executed in parallel aynway.

File: packages/schematics/update/update/npm.ts (hunk 2 above)
Comment (clydin) [reply packages/schematics/update/update/npm.ts]: not added yet.  wanted to make sure it was an adequate replacement first.

File: packages/schematics/update/update/npm.ts (hunk 1 above)
Comment (clydin) [reply packages/schematics/update/update/npm.ts]: Done.

**Pull request merged**